lighthouse-plugin-publisher-ads/audits/ad-render-blocking-resources.js

```diff
@@ -12,25 +12,35 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const common = require('../messages/common-strings');
 const NetworkRecords = require('lighthouse/lighthouse-core/computed/network-records');
 // @ts-ignore
 const RenderBlockingResources = require('lighthouse/lighthouse-core/audits/byte-efficiency/render-blocking-resources.js');
 const {auditNotApplicable} = require('../utils/builder');
-const {AUDITS, NOT_APPLICABLE} = require('../messages/messages');
 const {Audit} = require('lighthouse');
-const {formatMessage} = require('../messages/format');
 const {getPageStartTime} = require('../utils/network-timing');
 const {isGptTag} = require('../utils/resource-classification');
 const {URL} = require('url');
+// @ts-ignore
+const i18n = require('lighthouse/lighthouse-core/lib/i18n/i18n.js');
+
+const UIStrings = {
+  title: 'Minimal render-blocking resources found',
+  failureTitle: 'Avoid render-blocking resources',
+  description: 'Render-blocking resources slow down tag load times. Consider ' +
+  'loading critical JS/CSS inline or loading scripts asynchronously or ' +
+  'loading the tag earlier in the head. [Learn more](' +
+  'https://developers.google.com/web/tools/lighthouse/audits/blocking-resources' +
+  ').',
+  failureDisplayValue: 'Up to {opportunity, number, seconds} s tag load time ' +
+  'improvement',
+  columnUrl: 'Resource',
```

Comment:
Should column names go in common? I don't think we're consistent about naming them

Reply:
I think we should but agreed that we're not consistent. I'll do this in a followup PR. The word count difference for i18n should be trivial.

Reply:
https://github.com/googleads/publisher-ads-lighthouse-plugin/issues/102